test/merge-patient-lab-iips-mocha.js

```diff
@@ -1,174 +1,263 @@
 // merge-patient-lab-iips-mocha.js
 
 var chai = require('chai');
+
+var assert = chai.asert;
 var expect = chai.expect;
+
 var chaiAsPromised = require('chai-as-promised');
 chai.should();
 chai.use(chaiAsPromised);
 
+var sinon = require('sinon');
+
 var _ = require('underscore');
-var noflo = require('noflo');
 
 var compFactory = require('../components/merge-patient-lab-iips');
 
 var test = require('./common-test');
 var stubs = require('./common-stubs');
 
-describe('merge-patient-lab-iips', function() {
+describe( 'merge-patient-lab-iips', function() {
```

Comment:
It's no big deal to have, but since this modification is just adding the space I want to point out that these extra spaces are against our style guide AFAIK. See https://google.github.io/styleguide/cppguide.html#Formatting